postpic/particles/particles.py

```diff
@@ -473,14 +530,16 @@ def compress(self, condition, name='unknown condition'):
         """
```

Comment:
Maybe add to the docstring that, in contrast to `.add()`, this function returns a new object?

Reply:
done.